<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgDlgFileLayer</class>
 <widget class="QWidget" name="WdgDlgFileLayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>303</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item column="0" row="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Layer Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>txtLayerName</cstring>
       </property>
      </widget>
     </item>
     <item column="1" row="0">
      <widget class="QLineEdit" name="txtLayerName"/>
     </item>
     <item column="0" row="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>File:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>txtLayerName</cstring>
       </property>
      </widget>
     </item>
     <item column="1" row="1">
      <widget class="KisFileNameRequester" name="wdgUrlRequester" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblWarning">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Warning: Krita uses a relative path to store the location of the file in the .kra. If you move the .kra but not the file, the file layer may be broken.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpScalingOptions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Scaling Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item column="0" row="0">
       <widget class="QRadioButton" name="radioDontScale">
        <property name="text">
         <string>No Scaling</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item column="0" row="1">
       <widget class="QRadioButton" name="radioScaleToImageSize">
        <property name="text">
         <string>Scale to Image Size</string>
        </property>
       </widget>
      </item>
      <item column="0" row="2">
       <widget class="QRadioButton" name="radioScalePPI">
        <property name="text">
         <string>Adapt to Image Resolution (ppi)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisFileNameRequester</class>
   <extends>QWidget</extends>
   <header location="global">kis_file_name_requester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>